optuna/samplers/_search_space/intersection.py

```diff
@@ -33,7 +33,9 @@ def __init__(self, include_pruned: bool = False) -> None:
 
         self._include_pruned = include_pruned
 
-    def calculate(self, study: Study, ordered_dict: bool = False) -> Dict[str, BaseDistribution]:
+    def calculate(
+        self, study: Study, ordered_dict: bool = False, allow_empty_study: bool = True
```

Comment:
I'm not sure why this new argument is necessary for this PR. Could you elaborate on that?

Reply:
Sorry, I think I forgot to revert this part.